flink-runtime/src/main/java/org/apache/flink/streaming/api/datastream/KeyedStream.java

```diff
@@ -358,9 +359,10 @@ public <R> SingleOutputStreamOperator<R> process(
     @Internal
     public <R> SingleOutputStreamOperator<R> process(
             KeyedProcessFunction<KEY, T, R> keyedProcessFunction, TypeInformation<R> outputType) {
-
-        KeyedProcessOperator<KEY, T, R> operator =
-                new KeyedProcessOperator<>(clean(keyedProcessFunction));
+        OneInputStreamOperator operator =
```

Comment:
```suggestion
        OneInputStreamOperator<T, R> operator =
```

Maybe do similar for `flatMap` BTW